packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/stepper/default.example.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  extension,
+  Screen,
+  ScrollView,
+  Stepper,
+  Text,
+} from '@shopify/ui-extensions/point-of-sale';
+
+export default extension(
+  'pos.home.modal.render',
+  (root) => {
+    const mainScreen = root.createComponent(
+      Screen,
+      {name: 'stepper', title: 'Stepper'},
+    );
+    const scrollView =
+      root.createComponent(ScrollView);
+    const text = root.createComponent(
+      Text,
+      null,
+      '4',
+    );
+    const stepper = root.createComponent(
+      Stepper,
+      {
+        initialValue: 4,
+        onValueChanged: (value) => {
```

Comment:
Do we need to update the value prop of the stepper itself as well? If so @merkoyep and I did something similar for the TextField docs!

Reply:
https://shopify-dev.ui-extensions-70k7.nathan-oliveira.us.spin.dev/docs/api/pos-ui-extensions/unstable/components/stepper#stepper-propertydetail-value

No, in fact, the docs explain not to do this!

Reply:
I'm curious whats the difference between maintaining a external value state versus updating the internal state? Why is this not recommended @NathanJolly

Reply:
https://github.com/Shopify/pos-next-react-native/blob/main/src/components/Stepper/Stepper.tsx It's because this component is like... an uncontrolled component that also allows updates to state directly?

Reply:
Ah i see its explicitly stated in the component, thanks!